### PR TITLE
add mirroring action

### PR DESCRIPTION
**Pull request by JesusMcCloud**
This PR adds the mirroring action to this repo.
Please do check if more files should be pruned before we merge this into main. E. g. do we really want to build it and consume minutes on a mac runner. isn't testing enough?

Also: check the repository settings for the variables used in the action

### CODE REVIEW

File: .github/workflows/mirror.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (nodh) [inline .github/workflows/mirror.yml:1]: shouldn't we remove this comments?

**Review by nodh — CHANGES_REQUESTED** [review]

**Comment by nodh** [comment]
> This PR adds the mirroring action to this repo. Please do check if more files should be pruned before we merge this into main. E. g. do we really want to build it and consume minutes on a mac runner. isn't testing enough?

Probably we should remove every action, since they're running here anyways.

File: .github/workflows/mirror.yml (hunk 1 above)
Comment (JesusMcCloud) [reply .github/workflows/mirror.yml:1]: yes, i should

Comment (JesusMcCloud) [reply .github/workflows/mirror.yml]: removed

**Pull request merged**